FamilySearchIosSampleApp/LoginVC.swift

```diff
@@ -201,13 +201,17 @@ class LoginVC: UIViewController {
             let tabBarController : UITabBarController = (segue.destinationViewController as? UITabBarController)!
             tabBarController.tabBar.items![0].title = NSLocalizedString("tabAncestorsName", comment: "name for list tab")
             tabBarController.tabBar.items![1].title = NSLocalizedString("tabMemoriesName", comment: "name for memories tab")
-
-            let treeTVC : TreeTVC = (tabBarController.viewControllers![0] as? TreeTVC)!
+            
+            guard let treeTVC = tabBarController.viewControllers![0] as? TreeTVC else {
```

Comment:
You've got this in a `guard` so replace the `!` with a `?` like: `tabBarController.viewControllers?[0]`